en/engine/event/event-input.md

```diff
@@ -1,8 +1,13 @@
-# Global System Events
+# Input Event System
 
-In this section, the `global system events` of __Cocos Creator__ will be introduced.
+As mentioned in the previous document, `EventTarget` supports a complete set of event listening and emitting mechanism.  
+In Cocos Creator v3.4.0, we support the `input` object, which implements the event registering interface of `EventTarget`, and can register global system input events through this object. 
```

Comment:
```suggestion
Support for the `input` object, which implements the event registering interface of `EventTarget`, and can register global system input events through this object. 
```

Reply:
I removed Creator v3.4 as extraneous

Reply:
the `input` is supported on since v3.4.0, should we specify the engine version ?

Reply:
I change in this way https://github.com/cocos-creator/creator-docs/pull/1903/commits/e42e2c163f86896c0365159c133300dd3f5b9de8